### PR TITLE
2.2: Bump Marathon to 1.11.23

### PR DESCRIPTION
**Pull request by timcharper**
## Corresponding DC/OS tickets (required)

  - [MARATHON-8774](https://jira.mesosphere.com/browse/MARATHON-8774)
  - [MARATHON-8752](https://jira.mesosphere.com/browse/MARATHON-8752)
  - [MARATHON-8764](https://jira.mesosphere.com/browse/MARATHON-8764)

## Related tickets (optional)

  - [COPS-6599](https://jira.mesosphere.com/browse/COPS-6599)

### CODE REVIEW

**Review by pierrebeitz — APPROVED** [review]

**Comment by DominikDary** [comment]
@mesosphere-mergebot request-review @DominikDary

**Review by DominikDary — APPROVED** [review]

**Comment by timcharper** [comment]
@mesosphere-mergebot label-ready-for-review

File: CHANGES.md [hunk 1]
```diff
@@ -65,3 +65,9 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 * Enforce-role property no longer shows for sub-groups (MARATHON-8769)
 
 * Fix issue in which PUT /v2/groups ignored the enforceRole setting (MARATHON-8770)
+
+* Marathon can send placement constraint signals to Mesos in order to unusable offers (MARATHON-8764)
```
Comment (pierrebeitz) [inline CHANGES.md:69]: please rephrase!

File: CHANGES.md [hunk 2]
```diff
@@ -65,3 +65,9 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
...
+
+* Marathon will allow GROUP_BY placement constraints to be temporarily imbalanced during deployments. (MARATHON-8752)
+
+* Marathon no longer logs task data, which was a source of secrets for Docker containers to be logged. (MARATHON-8774)
```
Comment (pierrebeitz) [inline CHANGES.md:73]:
```suggestion
* Marathon no longer logs task data, as it may contain secrets for Docker containers. (MARATHON-8774)
```

**Review by pierrebeitz — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 2 above)
Comment (pierrebeitz) [reply CHANGES.md:73]: just a suggestion

File: CHANGES.md (hunk 1 above)
Comment (pierrebeitz) [inline CHANGES.md:69]:
```suggestion
* Marathon can send offer constraints to Mesos to reduce the number of offers Marathon needs to decline due to placement constraints (MARATHON-8764)
```

**Comment by pierrebeitz** [comment]
i'm currently waiting for the tests to finish. then i'd apply the suggestions to afterwards apply a `Ship It` right away.

**Comment by pierrebeitz** [comment]
@timcharper please have a look at my modifications to `Changes.md` and revert/edit if needed!

**Comment by pierrebeitz** [comment]
@mesosphere-mergebot label Ship It

**Comment by pierrebeitz** [comment]
applying ship it, as tests succeeded previously and we just updated `Changes.md` one more time.

**Pull request merged**